docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -166,15 +169,44 @@ public static void Example()
                 (elapsed2Ms / 1000).ToString() + " seconds");
         }
 
+        public static int LoadDataIntoBuffer(string path, ref VBuffer<Byte> imgData)
+        {
+            int count = -1;
+            // bufferSize == 1 used to avoid unnecessary buffer in FileStream
+            using (FileStream fs = new FileStream(path, FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize: 1))
```

Comment:
> bufferSize: 1)) [](start = 103, length = 16)

1? Can you please check why 1 is recommended?

Reply:
From what I understand, bufferSize = 1 will assign 1 byte of in-memory buffer for the FileStream. The possibility of GC is therefore reduced if we assign a smaller internal buffer. 
The suggestion is 4KB though.
https://stackoverflow.com/questions/3033771/file-i-o-with-streams-best-memory-buffer-size
Do you recommend I change it?

Reply:
https://source.dot.net/#System.Private.CoreLib/shared/Internal/IO/File.cs,fe0d7a82d3ee2a41

This is the format used in File.ReadAllBytes()